ReactNativeClient/root.js

```diff
@@ -660,6 +624,16 @@ class AppComponent extends React.Component {
 		});
 
 		AppState.addEventListener('change', this.onAppStateChange_);
+
+		const sharedData = await ShareExtension.data();
+		if (sharedData) {
+			reg.logger().info(`Received shared data: ${JSON.stringify(sharedData)}`);
```

Comment:
How large can the shared data be? Is there a chance stringifying a large object will crash the app? In any case, I think it would be safer to log minimal info here, for example just "Received shared data: title, body". Or even just "Received shared data".

Reply:
Fixed